Cargo.toml

```diff
@@ -22,21 +22,38 @@ arrow = { version = "*", default-features = false, optional = true}
 default = ["half"] # TODO: remove this as default feature as soon as https://github.com/CodSpeedHQ/codspeed-rust/issues/1 is fixed
 
 [dev-dependencies]
+# rstest = { version = "0.16", default-features = false}
+# rstest_reuse = "0.5"
 codspeed-criterion-compat = "1.0.1"
 criterion = "0.3.1"
 dev_utils = { path = "dev_utils" }
 
 
 [[bench]]
-name = "bench_f16"
+name = "bench_f16_return_nan"
+harness = false
+required-features = ["half"]
+
+# TODO: support this
+# [[bench]]
+# name = "bench_f16_ignore_nan"
+# harness = false
+# required-features = ["half"]
```

Comment:
Is currently not supported as we use the `ord_transform` to provide SIMD support for the non-hardware supported `f16` datatype (see #1)